.drone.yml

```diff
@@ -121,6 +130,9 @@ volumes:
 kind: pipeline
 name: manifest
 
+concurrency:
+  limit: 1
+
```

Comment:
This seems redundant on the last pipeline as it depends on the former 3 to be done. Is that so?

```suggestion
```

Reply:
I guess that would also work yes